docker-compose.yaml

```diff
@@ -5,15 +5,15 @@ services:
       target: production
     image: ffc-sfd-customer-receiver-payments
     container_name: ffc-sfd-customer-receiver-payments
-    depends_on:
-      - ffc-sfd-customer-receiver-payments-postgres
     environment:
       APPINSIGHTS_CLOUDROLE: ffc-sfd-customer-receiver-payments-local
       APPINSIGHTS_CONNECTIONSTRING: ${APPINSIGHTS_CONNECTIONSTRING}
-
-  ffc-sfd-customer-receiver-payments-postgres:
-    image: postgres:11.4-alpine
-    environment:
-      POSTGRES_DB: ffc-sfd-customer-receiver-payments
-      POSTGRES_PASSWORD: ppp
-      POSTGRES_USERNAME: postgres
+      MESSAGE_HOST: ${MESSAGE_HOST}
+      MESSAGE_USER: ${MESSAGE_USER}
+      MESSAGE_PASSWORD: ${MESSAGE_PASSWORD}
+      PAYMENTS_SUBSCRIPTION_ADDRESS: ${PAYMENTS_SUBSCRIPTION_ADDRESS}-${DEV_SUFFIX}
+      PAYMENTS_TOPIC_ADDRESS: ${PAYMENTS_TOPIC_ADDRESS}-${DEV_SUFFIX}
+      COSMOS_ENDPOINT: ${COSMOS_ENDPOINT}
```

Comment:
We need to add default values for these three cosmos_endpoint, cosmos_payments_database and cosmos_payments_container. Theyre not sensitive values so can be stored in the repo and means less env vars needed to run the service

Can find the syntax in one of the other repos 😄

Reply:
Great thank you for pointing this out. In that case will need to update this for the `ffc-sfd-customer-receiver-messages` service as well at some point.

Reply:
Yep, can create a ticket for it or do it on the next PR